makefile.shade

```diff
@@ -43,3 +44,7 @@ use-standard-lifecycle
   var splitPackagesExe = '${Directory.GetFiles(SPLIT_PROJECT_DOWNLOAD, "SplitPackages.exe", SearchOption.AllDirectories).First()}'
   var splitArgs = '--source ${BUILD_DIR} --csv ${SPLIT_CSV_PATH} --destination ${TARGET_DIR} --ignore-errors'
   exec program='${splitPackagesExe}' commandline='${splitArgs}' workingdir='${COHERENCE_BUILD_PROJ}'
+
+  var dependenciesPackagerExe = '${Directory.GetFiles(SPLIT_PROJECT_DOWNLOAD, "DependenciesPackager.exe", SearchOption.AllDirectories).First()}'
```

Comment:
Rename `SPLIT_PROJECT_DOWNLOAD` to `TOOLS_DOWNLOAD_PATH` or something like that.